lec17-rmarkdown.Rmd

```diff
@@ -1,4 +1,389 @@
+when preceded by '@' allows you to reference the citation in-line. They usually
+go within square brackets and are separated by semicolons.
+
+```
+One Citation:
+Some fact [@Smith2014]
+
+Multiple Citations:
+Statement [@Smith2014; @Logan1997].
+```
+
+To make a bibliography, you may also want to specify a citation style guide to
+format your bibliography (in the form of a .csl file). The official repository
+for recognized citation styles is available [here](https://github.com/citation-style-language/styles). Because of the permissive
+licensing, you can actually customize or make your own styles too! [This visual
+editor](http://editor.citationstyles.org/visualEditor/) is a great tool to modify
+styles to your liking.
+
+
+
+#### Exercise
+
+Take some time to find the style guide for your field and to download that file.
+
+Once you have the correct .csl file, you can specify it in your YAML metadata:
+
+```
+---
+title: "Sample Document"
+output: html_document
+bibliography: bibliography.bib
+csl: nature.csl
+---
+```
+
+### Footnotes
+
+```
+This is what a footnote looks like.[^1] Here is another.[^2]
+
+[^1]: My first footnote.
+[^2]: My second footnote.
+
+This will produce the following:
+```
+This is what a footnote looks like.[^1] Here is another.[^2]
+
+[^1]: My first footnote.
+[^2]: My second footnote.
+
+These are great because you can reference your footnotes by name and don't have to
+re-number them if things get reordered. The numbers in the rendered document will
+be reordered for you, by order of occurrence! From the [Pandoc documentation](http://pandoc.org/MANUAL.html#footnotes):
+
+> The identifiers in footnote references may not contain spaces, tabs, or newlines.
+> These identifiers are used only to correlate the footnote reference with the
+> note itself; in the output, footnotes will be numbered sequentially.
+
+### Output Formatting Options
+
+#### Changing the output file format
+
+To change the output format of your .Rmd file, try changing the `output`
+metadata in the YAML header from "html_document" to "word_document"
+
+#### Table of Contents
+
+To add a table of contents generated from the headers of your document, use
+the `toc` option as `true` and specify the depth of headers to list via
+`toc_depth` where the default is 3. These sections can also be numbered by
+using the `number_sections` option.
+
+```
+---
+title: "Making TOCs"
+output:
+    pdf_document:
+        toc: true
+        toc_depth: 2
+        number_sections: true
+---
+```
+
+#### Figure Options
+
+Some figure options can be set in the YAML header.
+
+- `fig_width` and `fig_height` can be used to control the default figure width
+and height (7x5 is used by default)
+- `fig_caption` controls whether figures are rendered with captions
+
+### Exercise
+
+1. Open up RStudio and make a new R Markdown file.
+2. Set up the YAML metadata:
+    - title is "Lecture 17 Exercise"
+    - author (you)
+    - output to word
```

Comment:
Would prefer HTML over Word, but I guess this might be needed for some journals.

Reply:
i guess in this case it might be best, since not everyone has word.

Reply:
(to output to html)

Reply:
or mention both? Since yes, some journals require Word .. .. . .